DEVELOPER_GUIDE.md

```diff
@@ -110,7 +110,7 @@ $ yarn osd clean
 
 OpenSearch Dashboards requires a running version of OpenSearch to connect to. In a separate terminal you can run the latest snapshot built using:
 
-_(Linux and Windows only - for MacOS, you'll need to [run OpenSearch from a tarball](#alternative---run-opensearch-from-tarball) instead)_
+_(Linux, Windows, Darwin only - for others, you'll need to [run OpenSearch from a tarball](#alternative---run-opensearch-from-tarball) instead)_
```

Comment:
We need similar updates on L187 and L217. For ease of use, should we say 
```suggestion
_(Linux, Windows, Darwin (MacOS) only - for others, you'll need to [run OpenSearch from a tarball](#alternative---run-opensearch-from-tarball) instead)_
```

I know `Darwin` !== `MacOS`, but contributors may not be aware of the fact that `Darwin` means it will work on their macbook.

Reply:
I was going to do that but glad someone else said it and not me! Ha. Updated!